Cargo.toml

```diff
@@ -44,3 +46,6 @@ tokio = { version = "1", features = ["rt-multi-thread", "macros", "fs", "io-util
 warp = "0.3.1"
 futures = "0.3.15"
 reqwest = "0.11.3"
+# Workaround to activate non-default features for tests:
+# https://github.com/rust-lang/cargo/issues/2911
+config = { path = ".", features = ["preserve_order"] }
```

Comment:
This may not want to be kept -- it's not a default feature to keep this an unbreaking change, but I wanted the tests to run with it enabled to prove it works

Reply:
I think it would be best to alter the actions step to simply use `test --all-features` and be done with it?

Reply:
Done :+1: